flink/v1.19/flink/src/main/java/org/apache/iceberg/flink/sink/shuffle/AggregatedStatisticsTracker.java

```diff
@@ -258,8 +258,7 @@ private AggregatedStatistics completedStatistics(long checkpointId) {
             sketch.getK(),
             sketch.getN(),
             sketch.getNumSamples());
-        return AggregatedStatistics.fromRangeBounds(
-            checkpointId, SketchUtil.rangeBounds(downstreamParallelism, comparator, sketch));
+        return AggregatedStatistics.fromKeySamples(checkpointId, sketch.getSamples());
```

Comment:
returning the complete samples from tracker. coordinator will store both complete samples and ranged bounds